docs/releases.rst

```diff
@@ -3,6 +3,133 @@
+
+**Configuration editing with patronictl**
+
+- Add configruation editing to patronictl (Ants Aasma, Alexander)
+
+  Add the ability to patronictl of editing dynamic cluster configuration stored in DCS. Support either specifying the
+  parameter/values from the command-line, involing the $EDITOR, or applying configuration from the yaml file.
+
+**Linux watchdog support**
+
+- Implement wachdog support for Linux (Ants)
+
+  Support Linux software watchdog in order to reboot the master node that losses the lock in Etcd. Normally, Patroni will
+  demote such a node right after figuring out it doesn't have a lock. The watchdog brings in an additional guarantee
+  that in the case of the Patroni crash the node will be rebooted and the database instance will not continue to run as a
+  master. It is possible to configure the watchdog device (`/dev/watchdog` by default) and the mode (on, automatic, off)
+  from the watchdog section of the Patroni configuration. You can get more information from the
+  :ref:`watchdog documentation<watchdog>`.
+
+**Add support for PostgreSQL 10**
+
+- Patroni is compatible with all beta versions of PostgreSQL 10 released so far and we expect it to be compatible with
+  the PostgreSQL 10 when it will be released.
+
+**PostgreSQL-related minor improvements**
+
+- Define pg_hba.conf via the Patroni configuration file or the dynamic configuration in DCS (Alexander)
+
+  Allow to define the contents of ``pg_hba.conf`` in the ``pg_hba`` sub-section of the ``postgresql`` section of the
+  configuration. This simplifies managing ``pg_hba.conf`` on multiple nodes, as one needs to define it only ones in DCS
+  instead of logging to every node, changing it manually and reload the configuration.
+
+  When defined, the contents of this section will replace the current ``pg_hba.conf`` completely. Patroni ignores it
+  if ``hba_file`` postgresql paramerer is set.
+
+- Support connecting via a unix socket to the local PostgreSQL cluster (Alexander)
+
+  Add the ``use_unix_socket`` option to the postgresql section of Patroni configuration. When set to true and the
+  Postgresql ``unix_socket_directories`` option is not empty, enables Patroni to use the first value from it to connect
+  to the local PostgreSQL cluster. If ``unix_socket_directories`` is not defined, Patroni will assume its default value
+  and omit the ``host`` parameter in the PostgreSQL connection string altogether.
+
+- Support change of superuser and replication credentials on reload (Alexander)
+
+- Support storing of configuration files outside of PostgreSQL data directory (@jouir)
+
+  Add the new configuration ``postgresql`` configuration directive ``config_dif``.
+  It defaults to the data directory and must be writable by Patroni.
+
+**Bug fixes and stability improvements**
+
+- Handle EtcdEventIndexCleared and EtcdWatcherCleared exceptions (Alexander)
+
+  Faster recovery when the watch operation is ended by Etcd by avoiding useless retries (Alexander)
```

Comment:
It's an explanation to the point immediately above. Duplicated a name by mistake.